src/ui/PageButton.tsx

```diff
@@ -8,21 +9,27 @@ export type PageButtonProps = {
 
 function PageButton(props: React.PropsWithoutRef<PageButtonProps>) {
   const { children, onClick, ...rest } = props;
+  const mainBgColor = useColorModeValue('ui.white', 'ui.black', 'ui.sepia');
+  const circleColor = useColorModeValue('ui.black', 'gray.700', 'ui.black');
   return (
-    <Flex alignItems="center">
+    <Flex alignItems="center" bgColor={mainBgColor}>
       <ChakraCircle
         as="button"
         d="flex"
         alignItems="center"
         border="1px solid"
         borderColor="ui.gray.light-cool"
-        color="ui.black"
+        color={circleColor}
         bg="inherit"
         size={{ sm: '40px', md: '50px' }}
         mr={8}
         ml={6}
         cursor="unset"
-        _hover={{ backgroundColor: 'gray.700', color: 'ui.white' }}
+        _hover={{
+          backgroundColor: 'gray.700',
+          color: 'ui.white',
+          border: 'none',
+        }}
```

Comment:
This is going to need to adapt to the color mode as well unfortunately

Reply:
Do you have a specific colors in mind for each mode? Or I can just set the current color for all of them for now and we edit it later.

Reply:
Honestly the button doesn't even need a hover style so we can just make it the same as the base style in each color mode